source/core/Math.cpp

```diff
@@ -294,8 +294,8 @@ DECLARE_SCALE2X_INTN_HELPER(Single)
     VIREO_FUNCTION_SIGNATURE7(InRangeAndCoerce##TYPE, TYPE, TYPE, TYPE, Boolean, Boolean, TYPE, Boolean) { \
```

Comment:
Done, but I only touched this code to fix warnings, the PR otherwise has nothing to do with it.